pyproject.toml

```diff
@@ -1,13 +1,84 @@
 [build-system]
 requires = [
-  "setuptools>=42",
-  "wheel",
-  "cmake>=3.21",
-  "conan==1.57.0",
-  "ninja",
-  "scikit-build==0.13.1",
+  "setuptools>=68",
+  "conan==1.61.0",
```

Comment:
This version differs from our conda setup. I needed to get a newer version here to make it work with scikit build. But conda still only has 1.59.

We need to update to version 2...